### PR TITLE
test: fixed

### PR DESCRIPTION
**Pull request by Scrum**
- fixed tests
- update dep dev
- add actions

I could deal with this project if you allow

### CODE REVIEW

**Comment by Scrum** [comment]
@eddyerburgh ping ?

**Comment by pearofducks** [comment]
Great job upgrading things!

A couple things that might need cleaned up:

- You're loading both `jsdom-global` and `browser-env`, and I don't think you need both.
- You added `window.Date = Date`, which I think is to work around a bug in `prettier`, but that really should have a comment so the reader knows why it's there.
- If there was a decision to keep `jsdom-global`, you'd need to add it to dependencies. Right now you're bringing the core `jsdom` package in via a transient dependency from `browser-env`.

Overall though it's awesome to see this modernized, and I'd love to see a new speed test now that Ava has seen a few major version bumps!

**Comment by Scrum** [comment]
@pearofducks Hi

> Overall though it's awesome to see this modernized, and I'd love to see a new speed test now that Ava has seen a few major version bumps!

I already did this, just forgot to do PR, thanks for reminding me)

Regarding `jsdom` I added comments and links to sources of the reason for the need to use this design

**Comment by pearofducks** [comment]
Looks good, can't `browser-env` be removed then? Or is it adding some functionality I don't know about?

**Comment by Scrum** [comment]
> Looks good, can't browser-env be removed then? Or is it adding some functionality I don't know about?

@pearofducks yep, see last commits )

**Comment by pearofducks** [comment]
Probably also need to add `jsdom` as a dependency. It was coming in via `browser-env` before.

**Comment by Scrum** [comment]
@pearofducks  Yes, you are absolutly right,  done.

**Comment by Scrum** [comment]
@pearofducks @eddyerburgh ping ?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Use Node.js 13.x
```
Comment (eddyerburgh) [inline .github/workflows/ci.yml]: We should use node 12 since it's the latest LTS.

File: package.json [hunk 2]
```diff
@@ -7,64 +7,68 @@
...
-    "eslint-config-standard": "^6.2.1",
-    "eslint-friendly-formatter": "^3.0.0",
-    "eslint-loader": "^1.7.1",
-    "eslint-plugin-html": "^3.0.0",
-    "eslint-plugin-promise": "^3.4.0",
-    "eslint-plugin-standard": "^2.0.1",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-transform-runtime": "^7.8.3",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/register": "^7.0.0",
```
Comment (eddyerburgh) [inline package.json]: @babel/runtime should be a dependency

**Comment by eddyerburgh** [comment]
Looks good, one small nitpick comment and I also need you to add @babel/runtime as a dev dependency.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Scrum) [reply .github/workflows/ci.yml]: ok, changed

File: package.json (hunk 2 above)
Comment (Scrum) [reply package.json]: fix, thanks

**Comment by eddyerburgh** [comment]
Thanks for making the changes. I just tested the `dev` script and it seems to be broken, could you also fix that and the `build` script?

**Comment by Scrum** [comment]
Oh sure

**Comment by Scrum** [comment]
@pearofducks @eddyerburgh done

**Comment by eddyerburgh** [comment]
Thanks. The dev script is fixed, but I'm still getting an error when I run the build script:

```Error: webpack.optimize.UglifyJsPlugin has been removed, please use config.optimization.minimize instead.```

**Comment by Scrum** [comment]
oh yes technology is developing rapidly) fixed

**Comment by eddyerburgh** [comment]
Tahnk you @Scrum 😄

**Pull request merged**